src/amchart/Column.js

```diff
@@ -34,6 +34,12 @@
     Column.prototype.publish("stackType", "regular", "set", "Stack Type",["none","regular","100%","3d"],{tags:["Basic"]});
     Column.prototype.publish("useOhlcLines", false, "boolean", "Use OHLC Lines",null,{tags:["Intermediate"]});
 
+    Column.prototype.publish("yAxisDomainLow", null, "number", "Y axis Minimum value",null,{optional:true});
+    Column.prototype.publish("yAxisDomainHigh", null, "number", "Y axis Maximum value",null,{optional:true});
+    Column.prototype.publish("yAxisBaseValue", null, "number", "Y axis base value",null,{optional:true});
+    Column.prototype.publish("yAxisLabelFrequency", 1,"number", "Y axis label frequency",null,{optional:true});
+    Column.prototype.publish("yAxisTickCount", 5,"number", "Y axis grid count",null,{optional:true});
```

Comment:
Do you want to default this to 5?  I would suggest leaving it as NULL as default?